.travis.yml

```diff
@@ -1,3 +0,0 @@
-language: java

```

Comment:
Why are you disabling travis completely? It would be better to actually enable running the tests on travis, so that for future PRs we will actually see whether they break existing tests.

Reply:
Actually, I need to set up Travis for the repo, currently, it is just a dummy script which is failing. So I removed it.

Reply:
@wmdietl, I have added travi-ci in #4